### PR TITLE
Indiquate Vagrant and VirtualBox supported version

### PR DESCRIPTION
**Pull request by Tom32i**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 ## Requirements
 
 * Make
-* [VirtualBox 5.0.20+](https://www.virtualbox.org/wiki/Downloads)
-* [Vagrant 1.8.4+](https://www.vagrantup.com/downloads.html)
+* [VirtualBox >=5.0.20 <5.0.28](https://www.virtualbox.org/wiki/Downloads)
+* [Vagrant 1.8.4](https://www.vagrantup.com/downloads.html)
```
Comment (chalasr) [inline README.md]: AFAIK `1.8.6` is buggy

Comment (ogizanagi) [inline README.md]: Any Vagrant version is buggy :trollface:

Comment (hyvs) [inline README.md]: 1.8.4 or 1.8.7+curl workaround ?

**Comment by benji07** [comment]
Spring cleaning, If this issue is still relevant, please reopen-it.

**Pull request closed without merging**